### ISSUE TITLE
linting typescript using eslint

### ISSUE THREAD

**Issue by aladdin-add** [survivejs/maintenance-book#95]
https://eslint.org/blog/2019/01/future-typescript-eslint

**Comment by sapegin** [comment]
I guess we should replace TSLint section with some notes on ESLint and TypeScript.

**Comment by bebraw** [comment]
Yeah, TSLint should go away and we should extend the TypeScript bits with notes on ESLint and how it goes together with it.
